### PR TITLE
Gentler text-protection scrim

### PR DESCRIPTION
**Pull request by mperrotti**
Using fancy new things like blend-modes on the gradient and CSS filters to help readability while making the scrim subtler.

Scrim falls back to just use `rgba` gradient for browsers that don't support this.

### CODE REVIEW

File: sass/ui-components/_toggle-pill.scss [hunk 1]
```diff
@@ -145,3 +145,45 @@ Checkbox form elements styled as toggle buttons.
 .toggleButton-icon {
 	margin-left: $space-quarter;
 }
+
+.quickJoin {
```
Comment (akdetrick) [inline sass/ui-components/_toggle-pill.scss]: should quickjoin be its own component?

File: sass/util/_type.scss [hunk 2]
```diff
@@ -68,12 +69,13 @@ $C_scrimGradient: #0F141F; // move to swatches?
 		width: 100%;
 		z-index: 0;
 		left: 0;
+    mix-blend-mode: hard-light;
```
Comment (akdetrick) [inline sass/util/_type.scss]: :rainbow:

File: sass/ui-components/_toggle-pill.scss (hunk 1 above)
Comment (mperrotti) [inline sass/ui-components/_toggle-pill.scss]: Ah crap, I accidentally included this in this PR. Remove anything in `_toggle-pill.scss` from this

Comment (akdetrick) [inline sass/ui-components/_toggle-pill.scss]: want me to just merge in the changes in these files?

- `sass/ui-components/_card.scss`
- `sass/util/_type.scss`

**Comment by akdetrick** [comment]
manually pulled in changes with `08f6a1c284bb132cdbb15c8b83f0c909f1162c90` in `release/2.0.0`

**Pull request closed without merging**